### PR TITLE
feat(table): adds support for routerlinks and svgIcons in table actions

### PR DESCRIPTION
**Pull request by schoenj**

### CODE REVIEW

**Review by 6rayWolf — APPROVED** [review]

**Comment by schoenj** [comment]
> I didn't do a full review, just took a quick glance over it. @Yirica should probably do the full review of this.

A review by @Yirica  has already been requested. ^^

File: projects/components/table/src/models.ts [hunk 1]
```diff
@@ -27,10 +27,17 @@ export const enum PsTableActionScope {
 export interface IPsTableAction<T> {
```
[the remark this answers is not in the document]
Comment (Yirica) [reply projects/components/table/src/models.ts:27]: Seems like a reasonable change to increase usability for other developers. I support this idea

**Comment by Yirica** [comment]
> 
> 
> I didn't do a full review, just took a quick glance over it. @Yirica should probably do the full review of this.

Why did I have to do another full review of this after @6rayWolf already approved it? Your comment would improve usability and I support that change, but nothing in this commit seems complicated enough to require two reviews? Just wondering

**Comment by saithis** [comment]
> > I didn't do a full review, just took a quick glance over it. @Yirica should probably do the full review of this.
> 
> Why did I have to do another full review of this after @6rayWolf already approved it? Your comment would improve usability and I support that change, but nothing in this commit seems complicated enough to require two reviews? Just wondering

Sorry, I just wanted to suggest you as a reviewer. But it seems like @6rayWolf was faster and approved it while I was writing that.

**Pull request merged**